docs/world api.md

```diff
@@ -1,95 +1,95 @@
-allows using WebSockets.
-
+Clients that connect to the server to sync items can be in any language that allows using WebSockets.
 
 ## Coding style
 
-AP follows [style.md](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/style.md).
-When in doubt use an IDE with coding style linter, for example PyCharm Community Edition.
-
+AP follows a [style guide](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/style.md).
+When in doubt, use an IDE with coding style linter, for example PyCharm Community Edition.
 
 ## Docstrings
 
-Docstrings are strings attached to an object in Python that describe what the
-object is supposed to be. Certain docstrings will be picked up and used by AP.
-They are assigned by writing a string without any assignment right below a
-definition. The string must be a triple-quoted string.
+Docstrings are strings attached to an object in Python that describe what the object is supposed to be. Certain
+docstrings will be picked up and used by AP. They are assigned by writing a string without any assignment right below a
+definition. The string must be a triple-quoted string, and should
+follow [reST style](https://peps.python.org/pep-0287/).
+
 Example:
+
 ```python
 from worlds.AutoWorld import World
+
+
 class MyGameWorld(World):
-    """This is the description of My Game that will be displayed on the AP
-       website."""
+    """This is the description of My Game that will be displayed on the AP website."""
 ```
 
-
 ## Definitions
 
-This section will cover various classes and objects you can use for your world.
-While some of the attributes and methods are mentioned here, not all of them are,
-but you can find them in `BaseClasses.py`.
+This section covers various classes and objects you can use for your world. While some of the attributes and methods
+are mentioned here, not all of them are, but you can find them in
+[`BaseClasses.py`](https://github.com/ArchipelagoMW/Archipelago/blob/main/BaseClasses.py).
 
 ### World Class
 
-A `World` class is the class with all the specifics of a certain game to be
-included. It will be instantiated for each player that rolls a seed for that
-game.
+A `World` is the class with all the specifics of a certain game that is to be included. A new instance will be created
+for each player of the game for any given generated multiworld.
 
 ### WebWorld Class
 
-A `WebWorld` class contains specific attributes and methods that can be modified
-for your world specifically on the webhost:
+A `WebWorld` class contains specific attributes and methods that can be modified for your world specifically on the
+webhost:
 
-`settings_page`, which can be changed to a link instead of an AP generated settings page.
+* `options_page` can be changed to a link instead of an AP-generated options page.
 
-`theme` to be used for your game specific AP pages. Available themes:
+* `theme` to be used for your game-specific AP pages. Available themes:
```

Comment:
I think something went wrong with the display here, it doesn't look right when selecting "View file"